docs/tasks/administer-cluster/kubeadm-upgrade-cmd.md

```diff
@@ -0,0 +1,203 @@
+[upgrade/staticpods] Waiting for the kubelet to restart the component
+[apiclient] Found 1 Pods for label selector component=kube-controller-manager
+[upgrade/staticpods] Component "kube-controller-manager" upgraded successfully!
+[upgrade/staticpods] Moved upgraded manifest to "/etc/kubernetes/manifests/kube-scheduler.yaml" and backuped old manifest to "/tmp/kubeadm-upgrade830923296/old-manifests/kube-scheduler.yaml"
+[upgrade/staticpods] Waiting for the kubelet to restart the component
+[apiclient] Found 1 Pods for label selector component=kube-scheduler
+[apiclient] Found 0 Pods for label selector component=kube-scheduler
+[apiclient] Found 1 Pods for label selector component=kube-scheduler
+[upgrade/staticpods] Component "kube-scheduler" upgraded successfully!
+[apiclient] Found 0 Pods for label selector k8s-app=self-hosted-kube-apiserver
+[apiclient] Found 1 Pods for label selector k8s-app=self-hosted-kube-apiserver
+[apiclient] Found 0 Pods for label selector k8s-app=self-hosted-kube-controller-manager
+[apiclient] Found 1 Pods for label selector k8s-app=self-hosted-kube-controller-manager
+[apiclient] Found 0 Pods for label selector k8s-app=self-hosted-kube-scheduler
+[apiclient] Found 1 Pods for label selector k8s-app=self-hosted-kube-scheduler
+[apiconfig] Created RBAC rules
+[addons] Applied essential addon: kube-proxy
+[addons] Applied essential addon: kube-dns
+```
+
+`kubeadm upgrade apply` does the following:
+
+- It checks that your cluster is in an upgradeable state, that is:
+  - The API Server is reachable,
+  - All nodes are in the `Ready` state, and
+  - The control plane is healthy
+- It enforces the version skew policies.
+- It makes sure the control plane images are available or available to pull to the machine.
+- It upgrades the control plane components or rollbacks if any of them fails to come up.
+- It applies the new `kube-dns` and `kube-proxy` manifests and enforces that all necessary RBAC rules are created.
+
+1. Manually upgrade your Software Defined Network (SDN).
+
+   Your Container Network Interface (CNI) provider might have its own upgrade instructions to follow now.
+   Check the [addons](/docs/concepts/cluster-administration/addons/) page to
+   find your CNI provider and see if there are additional upgrade steps
+   necessary.
+
+## Upgrading your worker nodes
+
+For each worker node (referred to as `$WORKER` below) in your cluster, upgrade `kubelet` by executing the following commands:
+
+1. Prepare the node for maintenance, marking it unschedulable and evicting the workload:
+
+```shell
+$ kubectl cordon $WORKER
+$ kubectl drain $WORKER
+```
+
+1. Upgrade the `kubelet` version on the `$WORKER` node, either by using a Linux distribution-specific package manager such as `apt-get` or `yum` or manually as described in the following:
+
+```shell
+$ sudo systemctl stop kubelet
+$ curl -s -L -o kubelet \
```

Comment:
This won't work, as the kubelet droping manifest also gets updated.
At this stage, run the full `apt-get upgrade` or `yum update` and write that here.
You may use tabs for this, look at the main kubeadm doc how to do so